spring-rabbit-test/src/test/java/org/springframework/amqp/rabbit/test/mockito/AnswerTests.java

```diff
@@ -32,47 +29,17 @@
  */
 public class AnswerTests {
 
-//	@Test
-//	public void testLambda() {
-//		Foo foo = spy(new Foo());
-//		doAnswer(new LambdaAnswer<String>(true, (i, r) -> r + r)).when(foo).foo(anyString());
-//		assertEquals("FOOFOO", foo.foo("foo"));
-//		doAnswer(new LambdaAnswer<String>(true, (i, r) -> r + i.getArguments()[0])).when(foo).foo(anyString());
-//		assertEquals("FOOfoo", foo.foo("foo"));
-//		doAnswer(new LambdaAnswer<String>(false, (i, r) ->
-//			"" + i.getArguments()[0] + i.getArguments()[0])).when(foo).foo(anyString());
-//		assertEquals("foofoo", foo.foo("foo"));
-//	}
-
 	@Test
 	public void testLambda() {
+		String property = System.getProperty("java.version");
+		System.out.println(property);
```

Comment:
hehe - I thought I'd removed these; they were from when I was debugging the IO compatibility build.